src/Sylius/Bundle/AddressingBundle/Form/Type/CountryChoiceType.php

```diff
@@ -41,18 +41,16 @@ public function configureOptions(OptionsResolver $resolver): void
             ->setDefaults([
                 'choice_filter' => null,
                 'choices' => function (Options $options): iterable {
-                    if (null === $options['enabled']) {
-                        $countries = $this->countryRepository->findAll();
-                    } else {
-                        $countries = $this->countryRepository->findBy(['enabled' => $options['enabled']]);
+                    if ($options['enabled'] === true) {
```

Comment:
```suggestion
                    if ($options['enabled']) {
```

Reply:
Actually I would stay with checking equality, because without it, it is also checking existance of the key, so for example
`'enabled' => false` will be in this case true